geth/transactions/utils.go

```diff
@@ -23,23 +21,24 @@ const (
 
 type contextKey string // in order to make sure that our context key does not collide with keys from other packages
 
-// All general log messages in this package should be routed through this logger.
-var logger = log.New("package", "status-go/geth/common")
+//ErrTxQueueRunFailure - error running transaction queue
+var ErrTxQueueRunFailure = errors.New("error running transaction queue")
 
-// ImportTestAccount imports keystore from static resources, see "static/keys" folder
-func ImportTestAccount(keystoreDir, accountFile string) error {
-	// make sure that keystore folder exists
-	if _, err := os.Stat(keystoreDir); os.IsNotExist(err) {
-		os.MkdirAll(keystoreDir, os.ModePerm) // nolint: errcheck, gas
-	}
+// HaltOnPanic recovers from panic, logs issue, sends upward notification, and exits
+func HaltOnPanic() {
```

Comment:
Do we need to export it? I would love to get rid of this function in the future so let's avoid it's used anywhere else.

Reply:
Nope, it's only used inside transactions package, I've also made private `MessageIDFromContext` and `Fatalf` as they were only used on transactions package too.